Gemfile.lock

```diff
@@ -525,4 +525,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.1.4
+   2.2.22
```

Comment:
Let's revert this line, as I'm not sure what impact this would have on downstream CI just yet

cc @jmartin-r7

Reply:
Correct at this time Metasploit's lock file needs to stay 2.1.x compatible since we share a single lock file for all platforms.